lib/interactive_viewer.ml

```diff
@@ -72,6 +109,7 @@ let view (patches : Patch.t list) =
   W.vbox
     [
       operation_info z_patches;
+      calculate_addition_removal_operation z_patches;
       current_operation z_patches;
```

Comment:
We have `operation_info`, `current_operation` and `current_hunks` here. The name `calculate_addition_removal_operation` is much longer and feels different: other names are descriptive of what they will display (they are ui components) while `calculate` seems to be an instruction. Try to make your name more similar to the one here! Ui component might need computation, but that's not how you describe them.